packages/grafana-ui/src/components/GraphNG/nullInsertThreshold.ts

```diff
@@ -0,0 +1,81 @@
+import { ArrayVector, DataFrame, FieldType } from '@grafana/data';
+
+export function nullInsertThreshold(frame: DataFrame, threshold?: number, refFieldName?: string): DataFrame {
+  const value = null;
+
+  if (frame.length < 2) {
+    return frame;
+  }
+
+  let refField = frame.fields.find((field) => {
+    // note: getFieldDisplayName() would require full DF[]
+    return refFieldName != null ? field.name === refFieldName : field.type === FieldType.time;
+  });
```

Comment:
i'm comfortable kicking this one down the road since `refFieldName` is not currently used at the call-sites. "first time field" is something we already do without erroring/handling when it's not the _only_ time field.